purpleair/channel.py

```diff
@@ -6,12 +6,20 @@
 from datetime import datetime, timedelta
 from typing import Optional
 
+try:
+    # py3
+    from urllib.parse import urlencode
+except ImportError:
```

Comment:
Remove this, the library does not target deprecated versions of Python.

Reply:
Removed py2 support